src/clr-angular/timeline/timeline-step-description.spec.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ClrTimelineStepDescription } from './timeline-step-description';
+
+export default function(): void {
+  describe('ClrTimelineStepDescription', () => {
+    describe('Template API', () => {
+      let fixture: ComponentFixture<TestTimelineStepDescription>;
+
+      let nativeElement: HTMLElement;
+
+      beforeEach(() => {
+        TestBed.configureTestingModule({
+          declarations: [ClrTimelineStepDescription, TestTimelineStepDescription],
+        });
+
+        fixture = TestBed.createComponent(TestTimelineStepDescription);
+        fixture.detectChanges();
+
+        nativeElement = fixture.nativeElement.querySelector('clr-timeline-step-description');
+      });
+
+      it('should add host classes', () => {
+        expect(nativeElement.className).toContain('clr-timeline-step-description');
+      });
+    });
+
+    describe('View', () => {
+      let fixture: ComponentFixture<TestTimelineStepDescription>;
+
+      let nativeElement: HTMLElement;
```

Comment:
Nit: move these definitions to top level so its not duplicated, same with beforeEach.